app/helpers/format-currency.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+const { Helper } = Ember;
+
+export function formatCurrency(params) {
+  let value = 0;
+  value = params[0];
+  let valueString = value.toString();
+  let dollars = Math.floor(value),
+      cents = (value - dollars).toFixed(2),
+      sign = '$';
```

Comment:
Allow the user to pass in a custom sign through the helper params. Let the default be `$`